examples/async-rest/async-rest-jar/pom.xml

```diff
@@ -11,7 +11,7 @@
   <name>Example Async Rest :: Jar</name>
   <url>http://www.eclipse.org/jetty</url>
   <properties>
-    <bundle-symbolic-name>${project.groupId}.examples.asyc.rest</bundle-symbolic-name>
+    <bundle-symbolic-name>org.eclipse.jetty.example.examples_async_rest</bundle-symbolic-name>
```

Comment:
`bundle-symbolic-name` is used only in OSGi manifest attributes, not in JPMS. The commit that introduced this change references JPMS, so perhaps it's wrong?

Furthermore, A) why `${project.groupId}` has been removed? and B) why has the name been changed from `examples.asyc.rest` to `example.examples_async_rest`? The name should remain the same.

All in all, I think this change is either a mistake, as I cannot see any reason for it?

Reply:
As we use ` <jpms-module-name>${bundle-symbolic-name}</jpms-module-name> ` I changed only ` <bundle-symbolic-name> ` but sure better only changing `  <jpms-module-name> `
Regarding the name change, two problems:
- typo asyc instead of async (so not matching the artifactId value)
- the groupId ` org.eclipse.jetty.example-async-rest ` is not valid value for ` Automatic-Module-Name ` because of ` - `

Reply:
@olamy nope.

`jpms-module-name` does not exist anymore, since we have now proper `module-info.java` files - although only for core modules, not for modules such as this one which is an example application.

Ok for the typo, but JPMS modules should have dots not underscores, so you need to revert this change back to what it was before (with the typo fixed).

Reply:
` pms-module-name ` is still used  ` <Automatic-Module-Name>${jpms-module-name}</Automatic-Module-Name> ` see  https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/pom.xml#L493